.github/workflows/publish_docs.yml

```diff
@@ -26,6 +26,10 @@ jobs:
     env:
```

Comment:
We now need to switch `branches` back to `main` I think. :) (GitHub won't let me comment on that line per se.)

Reply:
Yep, let's make that change (and any others) directly on `main` and just handle the merge here.

Reply:
Renaming in https://github.com/google/iree/pull/2336